service/storage_proxy.hh

```diff
@@ -480,7 +480,9 @@ public:
         return _db.local();
     }
 
-    db::view::update_backlog get_view_update_backlog() const;
+    db::view::update_backlog get_view_update_backlog_for_response() const;
+
+    std::optional<db::view::update_backlog> get_view_update_backlog_for_gossip(std::optional<db::view::update_backlog> last_backlog) const;
```

Comment:
I don't like these names because they don't really tell much about what those functions do. It's not obvious what are the requirements of "response" or "gossip".

Reply:
I changed the names a bit